ops/_tracing/export.py

```diff
@@ -0,0 +1,272 @@
+# Copyright 2022 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+# file except in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+"""FIXME Docstring."""
+
+from __future__ import annotations
+
+import contextlib
+import logging
+import os
+import ssl
+import threading
+import time
+import urllib.error
+import urllib.request
+from pathlib import Path
+from typing import Sequence
+
+# FIXME: single-file Python package can't be marked as py.typed
+import otlp_json  # type: ignore
+from opentelemetry.instrumentation.urllib import URLLibInstrumentor
```

Comment:
Ugh, I don't deeply nested packages like `opentelemetry.instrumentation.urllib`. In keeping with our style, what about a semi-consistent way to import these with a package name prefix, like:

```python
import opentelemetry.instrumentation.urllib as ot_urllib
from opentelemetry.sdk.resources as ot_resources
#
ot_urllib.URLLibInstrumentor()
...
```